src/style/index.less

```diff
@@ -12,6 +12,7 @@ body {
   z-index: 2;
   width: 100%;
   height: 100%;
+  overflow-x: visible !important;
```

Comment:
@devrchancay 
Esta línea está agregando un bug en mi testing.
Te dejo in gif :)